{{ cookiecutter.repo_name }}/.openshift/Jenkinsfile

```diff
@@ -234,6 +234,12 @@ void runInPrefectContainer(String env, String commands) {
     }
     docker.image("${BASE_IMG_NAME}:${prefect_version}").inside("-e PREFECT_API_URL=${prefect_api}") {
         sh """#!/bin/bash
+        python_path=\$(python -c "import sys; print(sys.executable.replace('bin/python', 'pip.conf'))")
+        echo [global] >> \$python_path
+        echo "index-url = http://do-prd-mvn-01.do.viaa.be:8081/repository/pypi-all/simple" >> \$python_path
+        echo "trusted-host = do-prd-mvn-01.do.viaa.be" >> \$python_path
+        pip install -r requirements.txt
```

Comment:
Moet dat in deze stap? 

Kan je ook zoiets doen?:
> pip3 install -r requirements.txt \
>     --extra-index-url http://do-prd-mvn-01.do.viaa.be:8081/repository/pypi-all/simple \
>     --trusted-host do-prd-mvn-01.do.viaa.be

Dan zijn de boenstaande commando's overbodig.

Reply:
Bij het builden van de deployment runt hij blijkbaar ook het script, of toch de imports ervan.